### PR TITLE
NO-ISSUE fix kubeAPI doc syntax highlighting

### PR DESCRIPTION
**Pull request by nmagnezi**

### CODE REVIEW

File: docs/Kube-API-examples.md [hunk 1]
```diff
@@ -84,11 +86,15 @@ In case of failure to apply the overrides the clusterdeployment conditions will
 
 Add an annotation with the desired options, the clusterdeployment controller will update the install config yaml with the annotation value.
 Note that this configuration must be applied prior to starting the installation
-```
-$kubectl annotate clusterdeployments.hive.openshift.io test-cluster -n assisted-installer adi.io.my.domain/install-config-overrides="{\"controlPlane\":{\"hyperthreading\":\"Disabled\"}}"
+```sh
+$kubectl annotate clusterdeployments.hive.openshift.io test-clusterhardwaredetails -n assisted-installer adi.io.my.domain/install-config-overrides="{\"controlPlane\":{\"hyperthreading\":\"Disabled\"}}"
```
Comment (rollandf) [inline docs/Kube-API-examples.md:90]: clusterhardwaredetails ?

File: docs/Kube-API-examples.md [hunk 2]
```diff
@@ -56,7 +58,7 @@ In case that the Bare Metal Operator is installed, the Baremetal Agent Controlle
     - Role: master/worker
     - Hostname (optional for user to set)
     - MachineConfigPool (optional for user to set)
-- Reconcile the BareMetalHost hardware details by copying the Agent's inventory data to the BMH's hardwaredetails annotation.
+- Reconcile the BareMetalHost hardware details by copying the Agent's inventory data to the BMH's hardware details annotation.
```
Comment (rollandf) [inline docs/Kube-API-examples.md:61]: It is called `"inspect.metal3.io/hardwaredetails"`, so maybe add quotes around it?

Comment (nmagnezi) [reply docs/Kube-API-examples.md]: looks okay now?

File: docs/Kube-API-examples.md (hunk 1 above)
Comment (nmagnezi) [reply docs/Kube-API-examples.md]: PBKAC.. (Done)

**Pull request merged**